packages/uilib/src/lib/plugins/plugin-store/plugin-store.svelte

```diff
@@ -0,0 +1,227 @@
+	.filter((plugin) => plugin.name !== 'Plugin Store')
+
+// #endregion
+
+// #region UI
+let notificationSnackbar: Snackbar
+
+let currentPluginAnchor: Element
+let menus: Menu[]
+$: menus = filteredPlugins.map(() => null)
+$: menuStates = filteredPlugins.map(() => false)
+
+function openPluginMenu(e, index: number) {
+	currentPluginAnchor = e.currentTarget
+	menuStates = menuStates.map(() => false)
+	menuStates[index] = true
+}
+
+// #endregion
+</script>
+
+<Dialog bind:open={isOpen} 
+fullscreen
+surface$style="width: 850px; max-width: calc(100vw - 32px);"
+aria-labelledby="plugin-store-title" 
+aria-describedby="plugin-store-content">
+<Header>
+    <Title id="plugin-store-title">Plugin Store</Title>
+    <IconButton class="material-icons" action="close">
+        <IconClose/>
+    </IconButton>
+</Header>
+<Content id="plugin-store-content">
+<Theme>
+    <plugin-store>
+        <plugin-store-toolbar>
+            <Checkbox label="Only Installed" bind:checked={showOnlyInstalled}/>
+            <Textfield
+            label={"Search"}
+            variant={"outlined"}
+            bind:value={searchFilter}
+            />
+        </plugin-store-toolbar>
+        <plugin-store-items> 
+            {#each filteredPlugins as plugin, index}
+            <plugin-store-item style={`background: rgba(0, 0, 0, ${index % 2 === 0 ? "0" : "0.03"});`}>
+                <plugin-store-item-meta>
+                <span style="font-size: 0.85rem;">Author</span>
+                <span><strong>{plugin.name}</strong></span>
+                <span style="font-size: 0.85rem;">https://sprinteins.github.io/plugins/{plugin.name.split(" ").join("-").toLowerCase()}/index.js</span>
+                </plugin-store-item-meta>
+                {#if plugin.installed } 
+                <Group variant="raised">
+                    {#if plugin.official}
+                        <Button on:click={() => togglePluginState(plugin, false) } variant="outlined" style="min-width: 148px;">
+                            <Label>Disable</Label>
+                        </Button>
+                    {:else}
+                        <Button on:click={() => togglePluginState(plugin, false)} variant="outlined" style="min-width: 102px;">
+                            <Label>Disable</Label>
+                        </Button>
+                        <div use:GroupItem>
+                            <Button on:click={(e) => openPluginMenu(e, index)} variant="outlined" style="min-width: 18px;">
+                                <IconArrowDropDown />
+                            </Button> 
+                            <Menu bind:this={menus[index]} anchorElement={currentPluginAnchor} open={menuStates[index]} anchorCorner="BOTTOM_LEFT" style="left: -70px">
+                                <List>
+                                    <Item on:SMUI:action={() => uninstallPlugin(plugin) }>
+                                        <Text>Uninstall</Text>
```

Comment:
Interesting that you can also uninstall the Plugin Store itself :) I don't see an issue with that, but think about if you want to keep it that way or if you want to restrict the user from doing that

Reply:
Haha I have thought about that as well, but no worries, that's not possible. Check the `filteredPlugins` variable. It doesn't get displayed in the list if it's the Plugin Store.

Reply:
Das meine ich

https://github.com/user-attachments/assets/3bdc8898-cba8-4bed-9623-e8d6b4cc9b30

Reply:
You have to name it Plugin Store when adding, as this will be the final name most likely. A hacky solution I know, but it gets the work done.